### PR TITLE
BaSP-MR-88: Set up GET method

### PR DESCRIPTION
**Pull request by SantiH97**
I implemented the Classes GET method using redux

### CODE REVIEW

**Comment by SantiH97** [comment]
[Classes GET method](https://trello.com/c/wOItZcM2/88-classes-get-method)

File: src/redux/classes/actions.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+};
+
+export const getClassesSuccess = (data) => {
+  return {
+    type: GET_CLASSES_SUCCESS,
+    payload: data
+  };
+};
+
+export const getClasseserror = (error) => {
```
Comment (Juanoauil) [inline src/redux/classes/actions.js:16]: check camelCase

File: src/Components/Classes/index.jsx [hunk 2]
```diff
@@ -1,47 +1,43 @@
...
-  const getClasses = async () => {
-    try {
-      const res = await fetch(`${process.env.REACT_APP_API_URL}/api/classes/`);
-      const data = await res.json();
-      setClasses(data.data);
-    } catch (error) {
-      console.error(error);
-    }
-  };
+  const classes = useSelector((state) => state.classes.classes);
```
Comment (DelfiLT) [inline src/Components/Classes/index.jsx:17]: You can use an object to not call one by one,` { classes, error, isLoading }`

**Review by Bigzufi — APPROVED** [review]
Good job with redux!

**Review by DelfiLT — APPROVED** [review]

**Review by keiler-art — COMMENTED** [review]
Great job!

**Review by Eliezer-Alberto — APPROVED** [review]
It's all fine!..

**Pull request merged**